hadoop-hdds/container-service/pom.xml

```diff
@@ -102,6 +102,10 @@
       <groupId>jakarta.xml.bind</groupId>
       <artifactId>jakarta.xml.bind-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>log4j</groupId>
+      <artifactId>apache-log4j-extras</artifactId>
+    </dependency>
```

Comment:
Since no Ozone code uses this, I think this is required only at runtime.  Thus, please add `<scope>runtime</scope>` (for each service).